src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PremisesEntity.kt

```diff
@@ -26,8 +26,21 @@ import javax.persistence.Table
 interface PremisesRepository : JpaRepository<PremisesEntity, UUID> {
   fun findAllByProbationRegion_Id(probationRegionId: UUID): List<PremisesEntity>
 
-  @Query("SELECT new uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.TemporaryAccommodationPremisesSummary(p.id, p.name, p.addressLine1, p.addressLine2, p.postcode, p.pdu, p.status, CAST(COUNT(b) as int)) FROM TemporaryAccommodationPremisesEntity p LEFT JOIN p.rooms r LEFT JOIN r.beds b GROUP BY p.id, p.name, p.addressLine1, p.addressLine2, p.postcode, p.pdu, p.status")
-  fun findAllTemporaryAccommodationSummary(): List<TemporaryAccommodationPremisesSummary>
+  @Query(
+    """
+        SELECT
+          new uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.TemporaryAccommodationPremisesSummary(p.id, p.name, p.addressLine1, p.addressLine2, p.postcode, p.pdu, p.status, CAST(COUNT(b) as int))
+        FROM
+          TemporaryAccommodationPremisesEntity p
+          LEFT JOIN p.rooms r 
+          LEFT JOIN r.beds b
+          LEFT JOIN p.probationRegion pr
```

Comment:
I should have sliced this up as 2 commits, 1 showing the refactor and 2 the addition. The new lines should be:

```
  LEFT JOIN p.probationRegion pr
WHERE 
  pr.id = :regionId
```